packages/client/modules/email/components/LimitsEmails/ThirtyDayWarningEmail.tsx

```diff
@@ -0,0 +1,88 @@
+import {ContactInfo, ExternalLinks} from 'parabol-client/types/constEnums'
+import React from 'react'
+import {EMAIL_CORS_OPTIONS} from '../../../../types/cors'
+import makeAppURL from '../../../../utils/makeAppURL'
+import {emailCopyStyle, emailLinkStyle} from '../../styles'
+import Button from '../Button'
+import EmailBlock from '../EmailBlock/EmailBlock'
+import EmailFooter from '../EmailFooter/EmailFooter'
+import EmptySpace from '../EmptySpace/EmptySpace'
+import Header from '../Header/Header'
+import Layout from '../Layout/Layout'
+import {LimitsEmailProps} from './LockedEmail'
+
+const innerMaxWidth = 480
+
+const copyStyle = {
+  ...emailCopyStyle
+}
+
+const linkStyle = {
+  ...emailCopyStyle,
+  ...emailLinkStyle
+}
+
+type Props = LimitsEmailProps & {
+  stickyTeamCount: number
+}
+
+export default function ThirtyDayWarningEmail(props: Props) {
+  const {appOrigin, preferredName, orgId, orgName, stickyTeamCount} = props
+  const billingURL = makeAppURL(appOrigin, `/me/organizations/${orgId}/billing`, {
+    searchParams: {
```

Comment:
@tghanken I'm adding these search params so that we can tell where the user is coming from when the land on the billing page. Is that what you have in mind? Do you know if we're tracking the URL params in the billing page yet? And would you like me to edit these params at all?

Reply:
I think these params make sense based on what we've collected historically.  We're collecting URL params globally throughout the site using Segment pageviews, so we should be setup to capture this!